src/Templates/NServiceBusDockerContainer/Program.cs

```diff
@@ -18,6 +18,15 @@ static IHostBuilder CreateHostBuilder(string[] args)
         {
             return Host.CreateDefaultBuilder(args)
                 .UseConsoleLifetime()
+                // TODO: NServiceBus.Extensions.Hosting comes with built-in logging support for Microsoft.Extensions.Logging
```

Comment:
Nitpick: Should we really mention the NSB option? Why would a customer want to not use the MS logging?

Reply:
I added it for completeness reasons because so far we haven't officially deprecated those, either way is fine for me

Reply:
Since this is a template I think we can choose what approach to promote?

Reply:
So your preference would be to remove the statement?

Reply:
Yes :)